### PR TITLE
Adding support for Flight Create Orders API

### PR DESCRIPTION
**Pull request by akshitsingla**
#36 Adding support for Flight Create Orders API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,6 +224,54 @@ amadeus.shopping.flightOffers.get({
     console.log(responseError);
 });
 
+// Flight Create Orders
+amadeus.shopping.flightOffersSearch.get({
```
Comment (tsolakoua) [inline README.md]: @akshitsingla I think in the README we don't need to have such an explicit example. For example in Python's README we have:
`# Flight Create Orders
amadeus.booking.flight_orders.post(flights[0], traveler)`

We do have a separate real [examples](https://github.com/amadeus4dev/amadeus-python/tree/master/examples) and the one for Flight Create Orders is in this [PR](https://github.com/amadeus4dev/amadeus-python/pull/68)

File: src/amadeus/namespaces/booking/flight_orders.js [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+            "countryCode": "ES"
+          }
+        }
+      ]
+    }
+  }));
+   * ```
+   */
+  post(params = {}) {
+    return this.client.post('/v1/booking/flight-orders', params);
```
Comment (tsolakoua) [inline src/amadeus/namespaces/booking/flight_orders.js]: Also, to help the user call the API easier, we build the POST body. Then the user has just to give as parameter the `flight` and `traveler` that are mandatory. Have a look at the implementation [here](https://github.com/amadeus4dev/amadeus-python/blob/master/amadeus/booking/_flight_orders.py).

**Comment by khanzzirfan** [comment]
Please review the PR for flight order api
https://github.com/amadeus4dev/amadeus-node/pull/53
@tsolakoua @akshitsingla

File: README.md (hunk 1 above)
Comment (akshitsingla) [reply README.md]: Fixed

File: src/amadeus/namespaces/booking/flight_orders.js (hunk 2 above)
Comment (akshitsingla) [reply src/amadeus/namespaces/booking/flight_orders.js]: As discussed, leaving a jsonBody as param for this method.

**Pull request merged**